srml/support/procedural/src/storage/genesis_config/genesis_config_def.rs

```diff
@@ -114,31 +116,40 @@ impl GenesisConfigDef {
 				.unwrap_or_else(|| quote!( Default::default() ));
 
 			config_field_defs.push(GenesisConfigFieldDef {
-				doc: line.doc_attrs.clone(),
 				name: config_field,
 				typ,
+				attrs: line.doc_attrs.clone(),
 				default,
 			});
 		}
 
 		for line in &def.extra_genesis_config_lines {
-			let doc = line.attrs.iter()
-				.filter_map(|a| a.parse_meta().ok())
-				.filter(|m| m.path().is_ident("doc"))
-				.collect();
+			let attrs = line.attrs.iter()
+				.map(|attr| {
+					let meta = attr.parse_meta()?;
+					if meta.path().is_ident("doc") || meta.path().is_ident("serde") {
```

Comment:
Can you please do it like I proposed? 

Next week the next one is coming and want to use feature xy for whatever reason. We just need to disallow `#[cfg()]` for now.